actions/README.md

```diff
@@ -0,0 +1,77 @@
+# Actions
```

Comment:
Let's also link this in handbook, if we already haven't.

Reply:
Let's put in chapter about communication UI <-> Provider

Reply:
I will merge now because we're going to use it on example project.